app/src/main/java/com/example/dailytaskmanager/MainActivity.java

```diff
@@ -0,0 +1,72 @@
+package com.example.dailytaskmanager;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import java.util.Calendar;
+
+import android.view.View;
+import android.widget.Button;
+import android.widget.DatePicker;
+import android.widget.EditText;
+import android.widget.Button;
+import android.widget.ImageButton;
+import android.widget.TimePicker;
+import android.app.DatePickerDialog;
+import android.app.TimePickerDialog;
+
+public class MainActivity extends AppCompatActivity implements View.OnClickListener{
+
+    ImageButton btnDatePicker, btnTimePicker;
+    EditText dateText, timeText;
+    private int mYear, mMonth, mDay, mHour, mMinute;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        btnDatePicker = (ImageButton) findViewById(R.id.dateButtonId);
+        btnTimePicker = (ImageButton) findViewById(R.id.timeButtonId);
+        dateText = (EditText) findViewById(R.id.dateDisplayId);
+        timeText = (EditText) findViewById(R.id.timeDisplayId);
+
+        Calendar c = Calendar.getInstance();

```

Comment:
A better practice in Java 8 is to use LocalDateTime. See https://stackoverflow.com/questions/5175728/how-to-get-the-current-date-time-in-java

Reply:
I have implemented LocalDateTime instead of Calendar Class